### PR TITLE
PhenoPLIER on Asthma-COPD Overlap Syndrome: add notebooks/scripts to run GLS model

### PR DESCRIPTION
**Pull request by miltondp**
This PRs adds notebooks to run the PhenoPLIER GLS model (Generalized Least Squares) to compute an LV-trait association for asthma, COPD and asthma/COPD overlap (ACO). Most of the code was **already reviewed**, and it is here used for these traits.

Other details to make reviewing easier:

* All Jupyter Notebooks (`.ipynb`) can be reviewed using the ReviewNB button below.
* Notebooks have also an equivalent `.py` file with plain Python code (stored in the `py/` subfolder).

### CODE REVIEW

File: projects/asthma-copd/nbs/30_gls_phenoplier/01-run_gls_phenoplier.ipynb [hunk 1]
```diff
@@ -0,0 +1,397 @@
+{
```
Comment (ivichadriana) [inline projects/asthma-copd/nbs/30_gls_phenoplier/01-run_gls_phenoplier.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #21.        GENE_CORR_FILE="${PHENOPLIER_PROJECTS_ASTHMA_COPD_RESULTS_DIR}/gls_phenoplier/gene_corrs/${pheno_id}/gene_corrs-symbols.per_lv"
</pre><p>Line is a but long, not sure if it's possible to split it though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: projects/asthma-copd/nbs/30_gls_phenoplier/05-summarize.ipynb [hunk 2]
```diff
@@ -0,0 +1,2293 @@
+{
```
Comment (ivichadriana) [inline projects/asthma-copd/nbs/30_gls_phenoplier/05-summarize.ipynb:1]: <p> </p><p>A comment at the top to broadly describe its function and how it is expected to be used (e.g. imported, run from command line, both).  &lt;&lt;&lt;&lt; from review checklist</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (ivichadriana) [inline projects/asthma-copd/nbs/30_gls_phenoplier/05-summarize.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    ):
</pre><p>unnecesary whitespace here maybe?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (ivichadriana) [inline projects/asthma-copd/nbs/30_gls_phenoplier/05-summarize.ipynb:1]: <p>I am not sure the dfs.head() output here is different from the one 4 cells above. If the point is to verify that the variables are categories, another way might be better?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

**Review by ivichadriana — APPROVED** [review]

File: projects/asthma-copd/nbs/30_gls_phenoplier/01-run_gls_phenoplier.ipynb (hunk 1 above)
Comment (miltondp) [reply projects/asthma-copd/nbs/30_gls_phenoplier/01-run_gls_phenoplier.ipynb:1]: <p>Yeah, I remember trying to split it before. Unfortunately, it doesn't seem to be very easy in bash.</p>

File: projects/asthma-copd/nbs/30_gls_phenoplier/05-summarize.ipynb (hunk 2 above)
Comment (miltondp) [reply projects/asthma-copd/nbs/30_gls_phenoplier/05-summarize.ipynb:1]: <p>Do you mean that the current description should be extended? The explanation on how to run the notebook (both from the browser and the command line) is in the <code>README.md</code> file in the root directory of the repository. I'm happy to add more details if needed!</p>

Comment (miltondp) [reply projects/asthma-copd/nbs/30_gls_phenoplier/05-summarize.ipynb:1]: <p>Yes, there are some aspects of the automatic code formatted that could be improved. Here I use <code><a href="https://github.com/psf/black" target="_blank">black</a></code> to automatically format the source code, which is PEP 8 compliant. So I just stick to whatever <code>black</code> suggests. </p>

Comment (miltondp) [reply projects/asthma-copd/nbs/30_gls_phenoplier/05-summarize.ipynb:1]: <p>Yes, you are right! I'll change this.</p>

Comment (miltondp) [reply projects/asthma-copd/nbs/30_gls_phenoplier/05-summarize.ipynb:1]: <p>Here what I'm using to check the types is the <code>dtypes</code> attribute, so I'll just use that again, and remove the extra call to <code>.head()</code> </p>

**Review by ivichadriana — COMMENTED** [review]
PR approved

**Pull request merged**